server/lib/message/message.handleEvent.js

```diff
@@ -8,6 +9,13 @@ const logger = require('../../utils/logger');
  */
 async function handleEvent(message) {
   try {
+    console.log(message);
+    this.event.emit(EVENTS.WEBSOCKET.SEND, {
+      type: WEBSOCKET_MESSAGE_TYPES.MESSAGE.SENT,
+      payload: message,
+      userId: message.user.id,
+    });
+    delete message.id;
```

Comment:
again, why do you do this? you're mutating an object in live that was already sent previously. 

This will lead to weird behavior, because object are sent as references in JS.

Reply:
After forum discussion, behavior has changed. This discussion is outdated.